src/main/java/it/pagopa/selfcare/pagopa/backoffice/config/JwtAspect.java

```diff
@@ -0,0 +1,61 @@
+package it.pagopa.selfcare.pagopa.backoffice.config;
+
+import it.pagopa.selfcare.pagopa.backoffice.exception.AppError;
+import it.pagopa.selfcare.pagopa.backoffice.exception.AppException;
+import it.pagopa.selfcare.pagopa.backoffice.security.JwtSecurity;
+import it.pagopa.selfcare.pagopa.backoffice.util.Utility;
+import lombok.extern.slf4j.Slf4j;
+import org.aspectj.lang.JoinPoint;
+import org.aspectj.lang.annotation.Aspect;
+import org.aspectj.lang.annotation.Before;
+import org.aspectj.lang.reflect.MethodSignature;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.stereotype.Component;
+
+@Aspect
+@Component
+@Slf4j
+public class JwtAspect {
+
+  private static final String LOCAL_ENV = "local";
+  private static final String TEST_ENV = "test";
+
+  private final String environment;
+
+  @Autowired
+  public JwtAspect(@Value("${info.properties.environment}") String environment) {
+    this.environment = environment;
+  }
+
+  @Before(
+      "within(@org.springframework.web.bind.annotation.RestController *) && @annotation(jwtSecurity)")
+  public void checkJwt(final JoinPoint joinPoint, final JwtSecurity jwtSecurity) {
+    var paramValue = getParamValue(joinPoint, jwtSecurity.paramName());
+
+    if (!this.environment.equals(LOCAL_ENV) && !this.environment.equals(TEST_ENV)) {
```

Comment:
What do we identify as a test environment? Does it include UAT as well? Is this exclusion necessary?